requirements.txt

```diff
@@ -1,3 +1,4 @@
+certifi==2018.10.15
```

Comment:
Y?

Reply:
`certifi.old_where()` were removed in the newer versions and this method is still used here: https://github.com/jobscore/sync-engine/blob/master/inbox/config.py#L18

Reply:
AFAIK this lib was the one causing troubles for upgrading the OS, we might want to give that a try again.